wasm/leb128/read_test.go

```diff
@@ -0,0 +1,24 @@
+package leb128
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestReadVarUint(t *testing.T) {
+	n, err := ReadVarUint(bytes.NewReader([]byte{0x80, 0x7f}))
+	if err != nil {
+		t.Fatal(err)
+	}
+	if n != uint32(16256) {
+		t.Fatalf("%d (expected) != %d", n, 16256)
```

Comment:
the canonical way to print these is:
```go
t.Fatalf("got=%d, want=%d", n, 16256)
```

or, even better, use a table-driven test:

```go
func TestReadVarUint(t *testing.T) {
	for i, test := range []struct {
		buf  []byte
		want uint32
	}{
		{
			buf:  []byte{0x80, 0x7f},
			want: 16256,
		},
	} {
		n, err := ReadVarUint(bytes.NewReader(test.buf))
		if err != nil {
			t.Errorf("%d: error reading var uint32: %v\n", i, err)
			continue
		}
		if n != test.want {
			t.Errorf("%d: got=%d. want=%d\n", i, n, test.want)
			continue
		}
	}
}
```

so adding new tests is easier.